Fixing #63

BaptisteVandecrux opened the pull request:
Steam point temperature was wrongly defined as `t_l +100` or `t_i +100`. 
Now it is defined as `T_0 + 100`.
Affects `rh_l_cor` and `rh_i_cor`.

PennyHow reviewed (dismissed):
Yes, a valid fix. And a very quick turnaround. Thank you!

I actually don't know if we need the deep copy of the `xr.Dataset` in each instance (`T_u`, `T_l`, `T_i`), but let's keep it as is for now.

PennyHow commented:
Ah, we have our process check failing in `L1toL2` on the following line:
```    
ds['rh_l_cor'] = correctHumidity(ds['rh_l'], ds['t_l'], T_l,           # Correct relative humidity
```

With the following message:
```
NameError: name 'T_l' is not defined
```

Because we are using `T_0` now, `T_l` and `T_i` are not defined. So a suggested correction is:

```    
ds['rh_l_cor'] = correctHumidity(ds['rh_l'], ds['t_l'], T_0,           # Correct relative humidity
```

And this will also be needed for correcting humidity from the instantaneous measurements also.

BaptisteVandecrux commented:
>  And a very quick turnaround. 

Maybe a bit too quick!
Anyway now I fixed the issues and removed the deep copies of T_u, T_l, T_i.
Can you r-make your comparison from issue #109? It will also make sure that the fix produces good values.
 
Great testing procedure!

PennyHow commented:
I'm very thankful for these testing procedures now! I'll plot it up and see whether it resolves issue #109

PennyHow commented:
I just checked the `rh_u` from NUK_K with this new fix. It doesn't seem to change anything from our side, but I can't see the same Edition 4 output as what Asiaq showed in #109.

So I think for our processing, this PR is fine now and running well. I'll get Asiaq to pull the latest data and see if that clears things up for them.

Our current processing of Edition 4 `rh_u_cor` vs. Edition 3 `RelativeHumidity(%)` 
![nuk_k_edition3_edition4_pre-fix](https://user-images.githubusercontent.com/28731816/223088295-6565c391-83f6-4b18-bf49-804f6fe8baa9.png)

This new developmental processing (ran with `Fixing-RH-correction`) `rh_u_cor` vs. Edition 3 `RelativeHumidity(%)`
![nuk_k_edition3_edition4_post-fix](https://user-images.githubusercontent.com/28731816/223088225-28890106-64bd-444f-a5cf-2c1f11dfe04a.png)

PennyHow commented:
Approved. Feel free to merge @BaptisteVandecrux
